services/lnd/grpc.js

```diff
@@ -196,10 +195,8 @@ class LndGrpcWrapper extends EventEmitter {
     const signData = getSignedDataFromMessage(message, receiverPubkey);
 
     const { signatureBytes: signature } = message;
-    const validMessage = await this.isValidMessage(signData, signature);
-    if (!validMessage) {
-      return null;
-    }
+    const validMessage =
+      (await this.isValidMessage(signData, signature)) || false;
```

Comment:
what's the purpose of `|| false` here?

Reply:
Hm it might not be necessary.  I was thinking that isValidMessage could return null and I wanted to be sure it was a boolean.  Can probably get rid of it because I think isValidMessage will return false.